packages/prisma/migrations/20220616090532_add_workflows/migration.sql

```diff
@@ -0,0 +1,73 @@
+-- CreateEnum
```

Comment:
Since these changes aren't in production yet, it's a good time to [squash all these migrations into a single one](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/squashing-migrations#migrating-cleanly-from-a-development-environment) before merging to prod.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix